sympy/concrete/products.py

```diff
@@ -331,6 +332,72 @@ def _eval_transpose(self):
             return self.func(self.function.transpose(), *self.limits)
         return None
 
+    def is_convergent(self):
+        """
+        See docs of Sum.is_convergent() for explanation of convergence
+        in SymPy.
+
+        The infinite product:
+
+        .. math::
+
+            \prod_{1 \leq i < \infty} f(i)
+
+        is defined to be the limit of the partial products
+
+        .. math::
+
+            f(1) f(2) \cdot \ldots \cdot f(n)
```

Comment:
This could perhaps be something like
``\prod_{i=1}^n f(i) = f(1)f(2)\cdots f(n)``
(I hope one backslash would suffice.)